site/docs/components/stack-layout/accessibility.mdx

```diff
@@ -0,0 +1,20 @@
+---
+# Leave the frontmatter as is
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+When building a sub-section of a page using Stack Layout, it is important to make sure you use semantic HTML, which means using the correct HTML elements for their intended purpose as much as possible.
+
+Stack Layout supports this behavior by allowing access to a prop called `as` that allows you to render the appropriate HTML instead of a plain `"div"`.
```

Comment:
```suggestion
Stack Layout supports this behavior by allowing access to a prop called `as` that allows you to render the appropriate HTML instead of a plain `div`.
```